cli/auth.go

```diff
@@ -83,6 +83,16 @@ type tokenResponse struct {
 	TokenType    string `json:"token_type"`
 }
 
+type authMethod interface {
+	fmt.Stringer
+	dialOpts() rpc.DialOption
+}
+
+var (
+	_ authMethod = (*token)(nil)  // Verify that *token implements authMethod.
```

Comment:
Definitely works/fine to leave as-is. Generally I just see `var _ authMethod = token{}` FWIW (or with an `&` if the type implements the interface with pointer receivers).